src/components/TimeSeries/TimeSeries.tsx

```diff
@@ -0,0 +1,25 @@
+import { useState } from 'react';
+
+import type { TimeSeriesData } from '../../types/models/timeSeries';
+import './TimeSeries.scss';
+import TimeSeriesChart from './TimeSeriesChart';
+import TimeSeriesForm from './TimeSeriesForm';
+
+function TimeSeries() {
```

Comment:
Just for consistency, I would add `export default` before the function declaration and not at the end of the file.